utils/create-component/createComponent.js

```diff
@@ -44,6 +43,12 @@ const questions = [
     message: 'Is this an unstable component (i.e. should it go in Canvas Kit Labs)?:',
     default: false,
   },
+  {
+    type: 'confirm',
+    name: 'publicModule',
+    message: 'Should this module be public?:',
```

Comment:
I think this question is a tad vague since this really relates only to publishing and not anything else. What we're really asking is do you want this module to be public upon publishing. Maybe add a brief explanation, something like:

`Publish this as a public module? (scoped modules are restricted access by default when published)`

I know it's a tad verbose but you really only skim through this prompt once and then you kind of get it later.

Reply:
Or maybe `Make access public when publishing?` since `Publish this as a public module?` may indicate that we're doing this right now.

Reply:
Sounds good. Will update. 👍